src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -138,48 +146,73 @@ const CardListItemIndex = ({
+            return null;
+        }
+    };
     return (
         <div data-test="card-list-item" className={classes.itemContainer}>
             <div className={classes.itemDataContainer}>
 
                 <div className={classes.itemValuesContainer}>
                     <Grid container >
-                        {
-                            profileImageDataElement &&
-                            <Grid item>
-                                {renderImageDataElement(profileImageDataElement)}
-                            </Grid>
-                        }
+                        <Grid item>
+                            {renderImageDataElement(profileImageDataElement)}
+                        </Grid>
                         <Grid item sm container>
                             <Grid item xs container direction="column" >
-                                {
-                                    dataElements
-                                        .map(({ id, name, type }) => (
-                                            <ListEntry
-                                                key={id}
-                                                name={name}
-                                                value={item.values[id]}
-                                                type={type}
-                                            />))
-                                }
-                                {
-                                    orgUnitName &&
-                                    <ListEntry name={i18n.t('Organisation unit')} value={orgUnitName} />
-                                }
-
-                                {
-                                    enrolledAt &&
-                                    // $FlowFixMe[prop-missing] automated comment
-                                    <ListEntry name={i18n.t('Date of enrollment')} value={enrolledAt} type={dataElementTypes.DATE} />
+                                {dataElements
+                                    .map(({ id, name, type }) => (
+                                        <ListEntry
+                                            key={id}
+                                            name={name}
+                                            value={item.values[id]}
+                                            type={type}
+                                        />))
                                 }
-
-                                {
-                                    programNameFromEnrollment &&
-                                    <ListEntry name={i18n.t('Program name')} value={programNameFromEnrollment} />
-                                }
-
+                                <ListEntry
+                                    name={i18n.t('Organisation unit')}
+                                    value={orgUnitName}
+                                />
+                                <ListEntry
+                                    name={i18n.t('Date of enrollment')}
```

Comment:
Does it make sense for this label to take the value of the metadata “Date of enrollment” configured from Maintenance? Thanks!

Reply:
thanks @simonadomnisoru good catch!